index.html

```diff
@@ -4,8 +4,16 @@
 
 <div class="container">
   <div class="row">
-    <div class="col-xs-3">
-      <p>Hello World</p>
+  	<div class="col-md-4">
+  		<h2>We are a community of young programmers taking charge of our education.</h2>
+  		<p>Pitt students of all majors and skill levels are welcome at our events.</p>
+  		<p>If you want to be a better programmer, you belong.</p>
+  		<img src="/assets/images/csc_selfie.jpg" class="img-responsive img-rounded" />
+  	</div>
+  	<div class ="col-md-1">
+  	</div>
+    <div class="col-md-7">
```

Comment:
Instead of putting an empty column between the first and last columns to leave one column width between them, you can add the `col-md-offset-1` class to the last column.